lib/liquid/standardfilters.rb

```diff
@@ -419,8 +419,8 @@ def at_most(input, n)
       result.is_a?(BigDecimal) ? result.to_f : result
     end
 
-    def default(input, default_value = ''.freeze)
-      if !input || input.respond_to?(:empty?) && input.empty?
+    def default(input, default_value = ''.freeze, false_as_missing = true)
+      if (false_as_missing ? !input : input.nil?) || input.respond_to?(:empty?) && input.empty?
```

Comment:
Could you add parentheses around the second part (so that I don't have to remember that `&&` is stronger than `||`)?